powershell/plugins/plugin-create-inline-properties.ts

```diff
@@ -43,6 +44,16 @@ function getNameOptions(typeName: string, components: Array<string>) {
   return [...result.values()];
 }
 
+function getCombinedDescription(rawDescription: string, externalDocsUrl?: string, externalDocsDescription?: string): string {
+  let description = rawDescription ?? "";
+  if (undefined !== externalDocsUrl && "" !== externalDocsUrl) {
+    description = description.concat(" Please visit external url ", externalDocsUrl, " to get more information.")
```

Comment:
A more common way to generate a format string is to use `${var}`. E.g. description =`Please visit external url ${externalDocsUrl} to get more information`